Source/GSXib5KeyedUnarchiver.m

```diff
@@ -276,6 +278,8 @@ + (void) initialize
                                            @"bordered", @"NSIsBordered",
                                            @"altersStateOfSelectedItem", @"NSAltersState",
                                            @"string", @"NS.relative",
+                                           @"canPropagateSelectedChildViewControllerTitle",
```

Comment:
The alternateName: method should have taken care of this name mapping. Why didn't it work?

Reply:
because the name of the property is NSTabViewControllerCanPropagateSelectedChildViewControllerTitle.  Does it handle adding the class name in front like that?  If so, I would be relieved to remove this.

Reply:
It does.